docker/scripts/app-rsync-extras.sh

```diff
@@ -33,33 +33,66 @@ done
 
 echo "Using destination $DEST_ROOT"
 
+EXCLUDE="$(mktemp)"
+cat << EOF > "$EXCLUDE"
+*#
+*%
+*.diff
+*.html
+*.json
+*.new
+*.p7s
+*.pdf
+*.ps
+*.tar
+*.xml
+*~
+/bcp
+/beta
+/fyi
+/ien
+/inline-errata
+/pending-errata
+/prerelease
+/std
+/v3test
+rfc[0-9]
+rfc[0-9][0-9]
+rfc[0-9][0-9][0-9]
+rfc[0-9][0-9][0-9][0-9]
+EOF
+
+OPTS="-asz --no-owner --no-group --partial ${PROGRESS:+--info=progress2} --exclude-from=$EXCLUDE --del --delete-excluded"
+
 for dir in bofreq; do
     dest="$DEST_ROOT/ietf-ftp/$dir"
     mkdir -p "$dest"
     echo "Fetching $dest ..."
-    rsync -auz ${PROGRESS:+--info=progress2} rsync.ietf.org::$dir/ $dest/
+    rsync $OPTS rsync.ietf.org::$dir/ $dest/
 done
 
 for dir in charter conflict-reviews internet-drafts review rfc slides status-changes yang; do
     dest="$DEST_ROOT/ietf-ftp/$dir"
     mkdir -p "$dest"
     echo "Fetching $dest ..."
-    rsync -auz ${PROGRESS:+--info=progress2} rsync.ietf.org::everything-ftp/$dir/ $dest/
+    rsync $OPTS rsync.ietf.org::everything-ftp/$dir/ $dest/
 done
 
 for dir in floor photo; do
     dest="$DEST_ROOT/media/$dir"
     mkdir -p "$dest"
     echo "Fetching $dest ..."
-    rsync -auz ${PROGRESS:+--info=progress2} rsync.ietf.org::dev.media/$dir/ $dest/
+    rsync $OPTS rsync.ietf.org::dev.media/$dir/ $dest/
 done
 
 dest="$DEST_ROOT/archive/id"
 mkdir -p "$dest"
 echo "Fetching $dest ..."
-rsync -auz ${PROGRESS:+--info=progress2} rsync.ietf.org::id-archive/ $dest/
+rsync $OPTS rsync.ietf.org::id-archive/ $dest/
 
 dest="$DEST_ROOT/www6s/proceedings"
 mkdir -p "$dest"
 echo "Fetching $dest ..."
-rsync -auz ${PROGRESS:+--info=progress2} rsync.ietf.org::proceedings/ $dest/
+rsync $OPTS rsync.ietf.org::proceedings/ $dest/
```

Comment:
Do we really need to sync all the proceedings?

Reply:
I don't think so. Do we need _any_?

Reply:
yes, it's where the minutes/agendas/slides are currently stored, so getting the most recent and upcoming meetings would likely be useful by default.

(We need to change where these are stored on disk in the long run)

Reply:
So one quick fix would be not not rsync proceedings of IETFs <100 (which is possible using the limited patterns that rsync allows) - should I add that?

Reply:
Try it and look at the size. I think you could get 11* easily too?